lib/matplotlib/tests/test_matplotlib.py

```diff
@@ -74,4 +75,6 @@ def test_importable_with__OO():
         "import matplotlib.patches as mpatches"
     )
     cmd = [sys.executable, "-OO", "-c", program]
-    assert subprocess.call(cmd, env={**os.environ, "MPLBACKEND": ""}) == 0
+    subprocess_run_for_testing(
+        cmd, env={**os.environ, "MPLBACKEND": ""}, check=True
+        )
```

Comment:
Also fits in a line, but maybe inline `cmd` instead?
```suggestion
    subprocess_run_for_testing(
        [sys.executable, "-OO", "-c", program],
        env={**os.environ, "MPLBACKEND": ""}, check=True)
```